<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modifys</class>
 <widget class="QDialog" name="Modifys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cheq_bu">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>***Seven Hundred Ninety Eight Only***</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acc_bu">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>ACCOUNT PAYEE ONLY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="amount_bu">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>**798.00**</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bearer_bu">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>XXXXXXXX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paynm_bu">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Payee Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="date_bu">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Day-Month-Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>28</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Bank Cheque Setting</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>621</width>
     <height>526</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Choose Bank To Modfiy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bank_ed"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Cheque_Width(cm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="cheq_w_sp"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Cheque_Height(cm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="cheq_h_sp"/>
      </item>
      <item>
       <widget class="QPushButton" name="create_bu">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>------------------------------------------------------------------------------------------------------------------------------------------------</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Bank</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bank_cb">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_bu">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ittm_nm_ed">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Left Margin(cm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="lm_spin"/>
      </item>
      <item>
       <widget class="QLabel" name="top_m_ed">
        <property name="text">
         <string>Top Margin(cm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="top_spin"/>
      </item>
      <item>
       <widget class="QLabel" name="width_ed">
        <property name="text">
         <string>Width(cm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="w_spin"/>
      </item>
      <item>
       <widget class="QLabel" name="height_ed">
        <property name="text">
         <string>Height(cm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="h_spin"/>
      </item>
      <item>
       <widget class="QPushButton" name="upload">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/res/template.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quit">
      <property name="text">
       <string>QUIT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label_9</zorder>
  <zorder>acc_bu</zorder>
  <zorder>amount_bu</zorder>
  <zorder>bearer_bu</zorder>
  <zorder>cheq_bu</zorder>
  <zorder>date_bu</zorder>
  <zorder>paynm_bu</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quit</sender>
   <signal>clicked()</signal>
   <receiver>Modifys</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>641</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
